src/ts/components/UIContainer.ts

```diff
@@ -99,13 +99,21 @@ export class UIContainer extends Container<UIContainerConfig> {
 
   private configureUIShowHide(player: PlayerAPI, uimanager: UIInstanceManager): void {
     let config = this.getConfig();
+    let isUiShown = false;
+
+    uimanager.onConfigured.subscribe(() => {
+      if (isUiShown) {
+        uimanager.onControlsShow.dispatch(this);
+      } else {
+        uimanager.onControlsHide.dispatch(this);
+      }
+    });
```

Comment:
We never dispatched the initial ui controls state in case a `hideDelay` is configured. This led to the issue that hiding/showing the UI didn't work properly.